config/public-api-v1.yml

```
@@ -31,31 +31,441 @@ paths:
+          maxLength: 3
+          example: 2FR
+        age_minimum:
+          type: integer
+          description: |
+            The minimum age of pupils this course is specified for.
+          example: 11
+        age_maximum:
+          type: integer
+          description: |
+            The maximum age of pupils this course is specified for.
+          example: 14
+        applications_open_from:
+          type: string
+          format: date
+          description: "Date from which applications can be submitted."
+          example: "2019-10-08"
+        bursary_amount:
+          type: integer
+          description: "Bursary amount for this course."
+          example: 9000
+        bursary_requirements:
+          type: string
+          description: "Description of requirements to be eligible for a bursary."
+          example: "a degree of 2:2 or above in any subject"
+        changed_at:
+          type: string
+          format: date-time
+          description: |
+            Date-time timestamp of when this course or any of it's related data changed.
+          example: "2019-06-13T10:44:31Z"
+        code:
+          type: string
+          description: |
+            Code that uniquely identifies this course within it's providers list of courses.
+          example: 3GTY
+          maxLength: 4
+        course_length:
+          type: string
+          description: |
+            Text describing how long the course runs.
+          example: "OneYear"
+        created_at:
+          type: string
+          format: date-time
+          description: "Timestamp of when this course was created."
+          example: "2019-06-13T10:44:31Z"
+        fee_details:
+          type: string
+          format: markdown
+          description: "Further details about the fees for this course, if applicable."
+          example: fee_details
+        fee_international:
+          type: integer
+          description: "Fee for international students (optional)."
+          example: 13000
+        fee_domestic:
+          type: integer
+          description: "Fee for UK and EU students."
+          example: 9200
+        financial_support:
+          type: string
+          format: markdown
+          description: "Details about financial support offered, if any."
+          example: financial_support
+        findable:
+          type: boolean
+          description: |
+            Is this course currently visible on the Find Postgraduate Teacher Training service.
+          example: true
+        funding_type:
+          type: string
+          description: |
+            The type of funding that maybe provided to candidates, if any.
+          example: apprenticeship
+          enum:
+            - ""
+            - salary
+            - apprenticeship
+            - fee
+        gcse_subjects_required:
+          type: string
+          example: [ maths, english ]
+          enum:
+            - [maths, english]
```

```suggestion
            - []
            - [maths, english]
```

Can we not have blank?